openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -679,11 +680,14 @@ def _validate_password(password, username=None, email=None, reset_password_page=
         (settings.ENABLE_AUTHN_RESET_PASSWORD_HIBP_POLICY and reset_password_page) or
         (settings.ENABLE_AUTHN_REGISTER_HIBP_POLICY and not reset_password_page)
     ):
-        pwned_response = check_pwned_password(password)
-        if pwned_response.get('vulnerability', 'no') == 'yes':
+        pwned_properties = check_pwned_password(password)
```

Comment:
i think we can keep the name `pwned_response`. That makes more sense to me.

Reply:
We are passing this response into segment event as properties that's why I renamed it to pwned_properties also we are already using this convention in other [places](https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/user_authn/tasks.py#L32).